com.intuit.graphql.parent/com.intuit.graphql.target/com.intuit.graphql.target.target

```diff
@@ -8,32 +8,37 @@
 			<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
-			<!-- newer Eclipse versions need Java 11 to run the Maven Tycho build -->
-			<repository location="https://download.eclipse.org/releases/2020-06"/>
+			<repository location="https://download.eclipse.org/releases/2023-03"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.12.0/"/>
+			<repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.14.0/"/>
```

Comment:
just curious if this should be 2.30.0 as well?

Reply:
this is different than xtext